common/network-common/src/main/java/org/apache/spark/network/server/RpcHandler.java

```diff
@@ -129,4 +137,38 @@ public void onFailure(Throwable e) {
 
   }
 
+  /**
+   * Handler for {@link MergedBlockMetaRequest}.
+   *
+   * @since 3.2.0
+   */
+  public interface MergedBlockMetaReqHandler {
+
+    /**
+     * Receive a {@link MergedBlockMetaRequest}.
+     *
+     * @param client A channel client which enables the handler to make requests back to the sender
+     *     of this RPC.
+     * @param mergedBlockMetaRequest Request for merged block meta.
+     * @param callback Callback which should be invoked exactly once upon success or failure.
+     */
+    void receiveMergeBlockMetaReq(
+        TransportClient client,
+        MergedBlockMetaRequest mergedBlockMetaRequest,
+        MergedBlockMetaResponseCallback callback);
+  }
+
+  /**
+   * A Noop implementation of {@link MergedBlockMetaReqHandler}.
```

Comment:
Could you add more description about the current usage of this Noop implementation in this PR?

Reply:
I have added where this Noop impl is used. PTAL. Let me know if I should describe it more.